### DISCUSSION TITLE
🔗 new dynamic tag : query result count (Duplicate of #22842)

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#23954] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

It would be great if we could display the number of posts per Loop Grid widget.

### Describe the solution you'd like

new dynamic tag for display query result count

**Comment by Joliamendi** [comment]
Excellent and useful 👌

**Comment by datgausaigon** [comment]
Suppose I have many Loop Grids in 1 Page, 3 Loop Grids A, B, C. A has 10 items, B has 12 items, C has 24 items. So is there this shortcode to count Results for each Loop Grid in a Page?

**Reply by 79ho3ein** [reply]
Of course, this dynamic tag has a section where we can determine which Loop widget to connect to. Just like the new taxonomy widget that has this section.

**Reply by codewizard13** [reply]
Also, Crocoblock JetEngine already has this feature and they allow to differentiate which filter / dynamic tag controls which loop via giving each loop widget an ID and assigning that ID as the "Query ID" value for the filter widget (or tag in this case). In JetEngine JetSmartFilters you can also say "I want these 3 post views to all be controlled by filter A" so then if you enter an address the map changes, the available hotel rooms changes, the nearby eateries changes, etc ... you get the idea.

But, this is definitely something I'd like to see implemented in Elementor natively

**Comment by nicholaszein** [comment]
Hey everyone!

Thank you for your contribution! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #22842 

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

🔍 Please @79ho3ein, in the future, remember to check if there isn't any thread similar to yours already opened. You have an habit of posting multiple duplicate issues and discussions, and it makes it very difficult to track them correctly and give them the appropriate priority.

Don't worry, we will count the votes in this discussion. 

Kind regards
